frontend/src/app/home/components/PerformanceReview.tsx

```
@@ -20,6 +20,7 @@ interface PerformanceReviewProps {
   onReviewResultsReceived: (
     review: Array<{ question: string; answer: string }>
   ) => void;
+  setIsReviewGenerating: (value: Boolean) => void;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)